AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/Migrations/20230227202735_Permissions.cs

```diff
@@ -0,0 +1,218 @@
+                table: "Settings",
+                column: "SettingGroupId",
+                principalTable: "SettingGroups",
+                principalColumn: "SettingGroupId",
+                onDelete: ReferentialAction.Cascade
+            );
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_Settings_SettingGroups_SettingGroupId",
+                table: "Settings"
+            );
+
+            migrationBuilder.DropTable(name: "Permissions");
+
+            migrationBuilder.DropTable(name: "SettingGroups");
+
+            migrationBuilder.DropTable(name: "Users");
+
+            migrationBuilder.RenameColumn(name: "SettingGroupId", table: "Settings", newName: "Id");
+
+            migrationBuilder.RenameIndex(
+                name: "IX_Settings_SettingGroupId",
+                table: "Settings",
+                newName: "IX_Settings_Id"
+            );
+
+            migrationBuilder
+                .AddColumn<string>(
+                    name: "BaseUserUserId",
+                    table: "Settings",
+                    type: "varchar(36)",
+                    nullable: true
+                )
+                .Annotation("MySql:CharSet", "utf8mb4");
+
+            migrationBuilder
+                .CreateTable(
+                    name: "BaseUsers",
+                    columns: table =>
+                        new
+                        {
+                            UserId = table
+                                .Column<string>(type: "varchar(36)", maxLength: 36, nullable: false)
+                                .Annotation("MySql:CharSet", "utf8mb4"),
+                            Email = table
+                                .Column<string>(type: "varchar(50)", maxLength: 50, nullable: false)
+                                .Annotation("MySql:CharSet", "utf8mb4"),
+                            Password = table
+                                .Column<string>(type: "varchar(36)", maxLength: 36, nullable: false)
+                                .Annotation("MySql:CharSet", "utf8mb4")
+                        },
+                    constraints: table =>
+                    {
+                        table.PrimaryKey("PK_BaseUsers", x => x.UserId);
+                    }
+                )
+                .Annotation("MySql:CharSet", "utf8mb4");
+
+            migrationBuilder.CreateIndex(
+                name: "IX_Settings_BaseUserUserId",
+                table: "Settings",
+                column: "BaseUserUserId"
+            );
+
+            migrationBuilder.AddForeignKey(
+                name: "FK_Settings_BaseUsers_BaseUserUserId",
+                table: "Settings",
+                column: "BaseUserUserId",
+                principalTable: "BaseUsers",
+                principalColumn: "UserId"
+            );
+        }
+    }
+}
```

Comment:
Hey sorry, should've called out this before, I said the same thing to Jaberul too, these files are auto-generated and you don't need to review them super closely. Everything in these migrations files is directly tied to the SettingsContext.cs file so that's all you'd really need to check. The same goes for the SQL files